jsonrpcbase/main.py

```diff
@@ -104,34 +104,37 @@ def call(self, jsondata, metadata=None):
         jsondata -- remote method call in jsonrpc format
         metadata -- optional additional data for the function call (eg. an auth token)
         """
-        result = self.call_py(jsondata, metadata)
+        try:
```

Comment:
I thought it made more sense to have an api where `.call` takes a json string, while `.call_py` takes python data. So I moved the json parsing and error handling into the `.call` method, which passes python data down to the `.call_py` method.